package.json

```diff
@@ -30,6 +30,7 @@
     "iflow-lodash": "^1.1.24",
     "iflow-react-router": "^1.2.1",
     "jest": "^20.0.4",
+    "jest-cli": "^20.0.4",
```

Comment:
Maybe this wasn't necessary, but tests failed for me with a module resolution error. Not finding jest-cli.